pkg/volume/cinder/cinder.go

```diff
@@ -503,7 +503,7 @@ func (c *cinderVolumeUnmounter) TearDownAt(dir string) error {
 	c.pdName = path.Base(refs[0])
 	klog.V(4).Infof("Found volume %s mounted to %s", c.pdName, dir)
 
-	// lock the volume (and thus wait for any concurrrent SetUpAt to finish)
+	// lock the volume (and thus wait for any concurrent SetUpAt to finish)
```

Comment:
/ok-to-test
/priority backlog
/lgtm

Reply:
/kind cleanup

Reply:
Hi @draveness, Thank you for there tags. It seems like 'tide' need the label approved to continue. Is this something you could do?

Reply:
No, please ping the approver for that